web/src/constants/contrastThemeMediaQueries.tsx

```diff
@@ -0,0 +1,5 @@
+export const contrastThemeMediaQueries = [
```

Comment:
🙃 Sorry, by extracting this to a shared constant I did not mean that you have to put it to an own file, only to reuse it for both usages in ´ThemeContext´. So feel free to move it back to `ThemeContext`.

Reply:
true :+1: